configurator/src/test/org/picmg/test/integrationTest/TestMaker/Test.java

```diff
@@ -18,41 +18,71 @@ public class Test {
         public String data;
         public String type;
         public String name;
-        public int delay = 25;
+        private int delay = 1000;
 
         public Step(String type, String id, String data)
         {
             this.id = id;
             this.data = data;
             this.type = type;
+            delay = 0;
+        }
+
+        public Step()
```

Comment:
I don't think this is a safe constructor. I see this is used and then given data with fromJson - why not make a constructor that takes the json object instead.